editor/editor_tests/user_stats_test.cpp

```diff
@@ -9,7 +9,8 @@ namespace editor
 namespace
 {
 auto constexpr kEditorTestDir = "editor-tests";
-auto constexpr kUserName = "Vladimir BI";
+// This user has made only 9 changes and then renamed himself, so there will be no further edits from him.
+auto constexpr kUserName = "Nikita Bokiy";
```

Comment:
Maybe I'm wrong, but 100% is in any case is better than "highly unlikely". Our tests should not  depend on OSM users whims. And if we will create a test user for our purposes, we will be able to test any behavior, as we can model any situation with this test account.